src/js/core/widget/core/Appbar.js

```diff
@@ -397,6 +404,28 @@
 				}
 			};
 
+			/**
+			 * Toggle visibility of BottomBar
+			 * @method _toggleSelectAll
+			 * @param {boolean} visible
+			 * @member ns.widget.core.Appbar
+			 * @protected
+			 */
+			prototype._toggleBottomBar = function (visible) {
+				this._ui.bottomBar.classList.toggle(classes.hidden, !visible);
+			};
+
+			/**
+			 * Method check if page has listview with checkbox in state checked
+			 * @method _isCheckedCheckbox
+			 * @member ns.widget.core.Appbar
+			 * @return {boolean}
+			 * @protected
+			 */
+			prototype._isCheckedCheckbox = function () {
+				return !!this._ui.page.querySelector(".ui-listview input[type='checkbox']:checked");
```

Comment:
It is recommended to check @HunseopJeong comments.
I agree with his opinion.

`".ui-listview input[type='checkbox']:checked" can select all checked input element in the listview.`
`".ui-listview > input[type='checkbox']:checked" seems to be more proper in this situation.`

Reply:
Done.

Reply:
Method has been removed.